Makefile

```diff
@@ -136,6 +137,17 @@ $(TEST): $(SOURCE) $(SPECS) $(BIN)
 testem:
 	$(BIN)/testem
 
+# Rules to attach checksums to JavaScript source in the header
+app.html: app.html.in .make-script-block
+	sed -e '/%SCRIPTS%/ {' -e 'r .make-script-block' -e 'd' -e '}' app.html.in > app.html
+
+.make-script-block: $(SOURCE)
+	> .make-script-block
+	for file in $(SOURCE); do \
+		checksum=$$(md5sum $$file | cut -c -32) ; \
```

Comment:
may i suggest changing this to

```sh
checksum=$$(md5sum $$file | awk '{ print $$1 }'); \
```

and bonus points if you can make this work on a Mac with `md5 -q` instead of `md5sum`